### ISSUE TITLE
Right Click open vscode proj folder from context menu "Open With"

### ISSUE THREAD

**Issue by DanielBox** [microsoft/vscode#141594]
Issue Type: <b>Feature Request</b>

I want to be able to right click a project folder and open folder with VS Code in Windows 11, like I could in Windows 10

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:40:02.816Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

**Comment by joeljose350** [comment]
It is possible in Windows 11 too. All you have to do is select the checkbox to activate this option while installing VS Code.

![image](https://user-images.githubusercontent.com/53969297/151324852-9fe61e40-dda6-414f-81ee-4ebb2f4032ce.png)

**Comment by lavadaragon15396** [comment]
> That option is available in Windows 11. All you have to do is select the checkbox to activate it while installing VS Code.
> 
> ![image](https://user-images.githubusercontent.com/53969297/151324852-9fe61e40-dda6-414f-81ee-4ebb2f4032ce.png)

No they mean the windows 11 context menu, 
like windows terminal does here: 
![image](https://user-images.githubusercontent.com/68852782/151398296-058f5261-3178-4597-8209-f5909cba75e0.png)

**Comment by DanielBox** [comment]
> 

Yeah I can get there with two steps by first choosing 'Show more options'. In windows 10 I"m pretty sure it was one step. It was on the first menu. Not a super big deal but I got accustomed to the routine. Thank you both for your replies.

**Comment by DanielBox** [comment]
In case it helps anyone I ended up just disabling the "show more options"  option and gong back to the old win 10 menu using the method outlined here: https://www.elevenforum.com/t/disable-show-more-options-context-menu-in-windows-11.1589/

**Comment by togetogejizou** [comment]
I heard that it can be achieved by using something called `Sparse Manifests`.
https://blogs.windows.com/windowsdeveloper/2021/07/19/extending-the-context-menu-and-share-dialog-in-windows-11/

Actual example of adding an item to the menu.
https://github.com/M2Team/NanaZip

**Comment by sbatten** [comment]
this is a dupe and shipped
